ghcide/src/Development/IDE/Plugin/Completions.hs

```diff
@@ -127,6 +127,7 @@ dropListFromImportDecl iDecl = let
     in f <$> iDecl
 
 resolveCompletion :: ResolveFunction IdeState CompletionResolveData Method_CompletionItemResolve
+resolveCompletion _ide _pid comp _uri NothingToResolve = pure comp
```

Comment:
comment?

Reply:
I thought the comment for `NothingToResolve`suffices, but will add the comment here as well.